### DISCUSSION TITLE
Can't query auth.users table

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#2575] [Questions] [answered]
Now I'm facing an issue when trying to access the users.auth table through the UI. A 504 error. 

Request URL: https://app.supabase.io/api/auth/lutcrhnocwappsgimbuo/users?limit=10&offset=0&keywords=

{"error": {"code": "504", "message": "An error occurred with your deployment"}}

**Comment by dshukertjr** [comment]
<strike>Hi @milestones95!

May I ask why you want to query the `auth.users` table? Is this for some batch action on the server side or something? 

Typically since `auth.users` table contains sensitive information, you don't want to query it. If you want to list the users safely, creating a table that contains users' information in `public` schema might be a good route to take! You can find examples [here](https://supabase.io/docs/guides/auth/managing-user-data)</strke>

Edited: I obviously did not read the question carefully enough 😂 +1 on silentworks's answer!

**Comment by silentworks** [comment, marked as answer]
Try logging out and logging in again, if the issue persists inside the Supabase UI then contact the Supabase team.
